src/replication_misc.c

```diff
@@ -310,3 +312,19 @@ set_socket_keepalive(int sfd)
 out:
 	return (ret);
 }
+
+json_object *
+json_object_new_uint64(uint64_t value)
+{
+	/* 18446744073709551615 */
+	char num[21];
+	int r;
+	json_object *jobj;
+
+	r = snprintf(num, sizeof(num), "%" PRIu64, value);
+	if (r < 0 || (size_t)r >= sizeof(num))
```

Comment:
why size_t? it should be just r >= sizeof(num)

Reply:
`r` is defined as `int` type. So for comparison, we are doing typecast.